src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -2033,4 +2089,30 @@ public function get_transaction_url( $order ) {
 		return parent::get_transaction_url( $order );
 	}
 
+    /**
+     * Isolates static addNotice calls.
+     *
+     * @param  string $message
+     * @param  string $type
+     */
+    protected function staticAddNotice($message, $type)
+    {
+        Mollie_WC_Plugin::addNotice(
+            __(
+                $message,
+                $type
+            )
```

Comment:
You cannot pass variables to those functions, them will not be translated by poedit or glotpress. 
Call directly the function over the strings to make them translatable, also the textdomain (which is not type).
Check this article http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/ if you want more information about how to translate correctly.